nilearn/plotting/surf_plotting.py

```diff
@@ -369,7 +384,8 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
                        hemi='left', view='lateral', threshold=None,
                        alpha='auto', vmax=None, cmap='coolwarm',
                        symmetric_cbar="auto", bg_on_data=False, darkness=1,
-                       title=None, output_file=None, **kwargs):
+                       title=None, output_file=None, ax=None, fig=None,
```

Comment:
same here ax -> axes, fig-> figure just to be naming consistent with other volume plotting functions.